project.go

```diff
@@ -272,20 +272,25 @@ func (h *ProjectsHandler) GetEventLog(project string) ([]*ProjectEvent, error) {
 	return r.Events, errR
 }
 
-// ServiceTypes returns all the available service types
-func (h *ProjectsHandler) ServiceTypes(project string) (map[string]ServiceType, error) {
+// ServiceTypes returns all the available service types.
+func (h *ProjectsHandler) ServiceTypes(project string, f Filter) (map[string]ServiceType, error) {
```

Comment:
We may have move filters in the future, perhaps it is going to be `[]Filter` in the next iteration I think. Nothing to worry about right now.

Reply:
this is no longer relevant